### ISSUE TITLE
[Enhancement]: add a search button that can be pressed

### ISSUE THREAD

**Issue by Charles-Em** [C9Glax/tranga-website#86]
### What is broken?

Hello, 

I'm wondering if there could be a button to launch a search
Currently, on my computer I simply press enter.
For I haven't found a way to do it when on my phone, and I would love to be able to add a manga from my couch.
Think it could be added?

Charles

**Comment by C9Glax** [comment]
Moved to correct repo. @db-2001  this one's for you

**Comment by db-2001** [comment]
Yeah, good call. There's a lot of work to be done to make the front-end mobile friendly.

**Comment by C9Glax** [comment]
#159

**Comment by C9Glax** [comment]
Will be included in vite-react-ts (Server-V2)
